app/models/notification.rb

```diff
@@ -51,33 +51,7 @@ def send_to_followers(notifiable, action = nil)
     def send_new_comment_notifications(comment)
       return if comment.commentable_type == "PodcastEpisode"
 
-      user_ids = comment.ancestors.select(:user_id).where(receive_notifications: true).pluck(:user_id).to_set
-      user_ids.add(comment.commentable.user_id) if comment.commentable.receive_notifications
-      json_data = {
-        user: user_data(comment.user),
-        comment: comment_data(comment)
-      }
-      user_ids.delete(comment.user_id).each do |user_id|
-        Notification.create(
-          user_id: user_id,
-          notifiable_id: comment.id,
-          notifiable_type: comment.class.name,
-          action: nil,
-          json_data: json_data,
-        )
-        # Be careful with this basic first implementation of push notification. Has dependency of Pusher/iPhone sort of tough to test reliably.
-        send_push_notifications(user_id, "@#{comment.user.username} replied to you:", comment.title, "/notifications/comments") if User.find_by(id: user_id)&.mobile_comment_notifications
-      end
-      return unless comment.commentable.organization_id
-
-      Notification.create(
-        organization_id: comment.commentable.organization_id,
-        notifiable_id: comment.id,
-        notifiable_type: comment.class.name,
-        action: nil,
-        json_data: json_data,
-      )
-      # no push notifications for organizations yet
+      Notifications::NewComment::Send.call(comment)
     end
     handle_asynchronously :send_new_comment_notifications
```

Comment:
shouldn't this be removed and the `NewCommentJob` used? 

Because otherwise it seems you're effectively using `NewCommentJob` only in the spec, am I wrong?